src/sfdt/__tests__/replaceSfdtBookmarks.test.js

```diff
@@ -0,0 +1,99 @@
+import get from 'lodash/get';
+import {getSFDT, getInline, getInlines} from '../../__tests__/utils';
+import tableInlines from './fixtures/tableInlines';
+
+import replaceSfdtBookmarks from '../replaceSfdtBookmarks';
+
+const data = {
+	K1: '123'
+};
+
+const data1 = {
+	K2: 'def'
+};
+
+const inlines = getInlines();
+const sfdt = getSFDT(inlines);
+
+// test for field data
+describe('sfdt bookmark modify', function() {
+	test('replaceSfdtBookmarks', function() {
+		// calculate length of inlines array
+		const originalLength = getInline(sfdt).length;
+
+		// run function we are testing
+		const result = replaceSfdtBookmarks(data, sfdt);
+		const currentInlines = getInline(result);
+
+		// make sure we get the same length of inlines, no addition, no subtraction
+		expect(currentInlines.length).toEqual(originalLength);
+		// check replacement went well
+		expect(currentInlines[1].name).toEqual('DATA::123');
+		expect(currentInlines[3].name).toEqual('DATA::123');
+	});
+
+	test('when data already exists', function() {
+		const sfdtWithData = {
```

Comment:
You can use getSFDT function here

Reply:
@ankitaShrma it's same as in populate. both works same. im using this instead of using getSFDT function bcoz it will be easy to insert some data in sfdt in this case. (check line 48)